03/app.test.js

```diff
@@ -1 +1,21 @@
-import randomNumber from './app';
\ No newline at end of file
+import randomNumber from './app';
+test('randomNumber should return a number', () => {
+  expect(typeof randomNumber(1, 10)).toBe('number');
+});
+test('randomNumber should throw an error if argument is not a number', () => {
+    expect(() => randomNumber('1', 10)).toThrow();
```

Comment:
Warto wskazać jakiego błędu się spodziewamy, bo tylko wtedy mamy pewność, że to ten, który testujemy tj. `.toThrow('message');`